### ISSUE TITLE
/vagrant won't mount properly, with "Smart Mount" enabled when vagrant.pkg is still mounted

### ISSUE THREAD

**Issue by datfinesoul** [Parallels/vagrant-parallels#124]
When Parallels is configured use "Smart Mount" (which seems to be the default for me), and if the `vagrant.pkg` is still open/mounted, Vagrant will mount both `/vagrant` to the startup dir, and `/Vagrant` to the pkg dir.  But in Linux, (Ubuntu 13.10 at least), the `/vagrant` folder will show the files from the pkg mount.

It's somewhat of an edge case, but I found it really hard to find the issue from the logs and wasn't sure if the problem was with parallels, or vagrant-parallels.

Is it possible to disable "Smart Mount" from the Vagrantfile or something similar to prevent this from happening?

**Comment by silentroach** [comment]
@legal90, here is the way to disable smartMount

```ruby
parallels.customize ["set", :id, "--smart-mount", "off"]
```

**Comment by legal90** [comment]
@silentroach Yup, exactly. The full answer will be:
```
config.vm.provider "parallels" do |v|
  v.customize ["set", :id, "--smart-mount", "off"]
end
```

**NB!** It works only with Parallels Desktop >= 10.1.2 

P.s. I'll try to make this setting default prior to mark this issue as resolved

**Comment by legal90** [comment]
Fixed in [v1.5.0](https://github.com/Parallels/vagrant-parallels/releases/tag/v1.5.0)
Option `--smart-mount` is set to `off` for Parallels Desktop >= 10.1.2
